product/README.md

```diff
@@ -0,0 +1,74 @@
+# Product @ DVELP
+Our Playbook for creating products at DVELP.
+
+## Our Mission
+To create solutions that help businesses revolutionise how they speak with
+their customers.
+
+## Strategy
+To [identify White Space](#fill-the-white-space) in [Partner Solutions](/partner-solutions),
+[Prototype rapidly](#get-close-with-customers) and validate hypothesis amongst
+our customer base. Pick winners early and [scale up](#piggyback-on-partners)
+through strong vendor partner relationships.
+
+### Get Close with Customers
+Through close relationships with customers we can deliver new features and
+prototypes to rapidly validate hypotheses.
+
+### Fill the White Space
+Leverage insight from customers to recognise industry trends early and identify
+opportunities to fill `White Space` left by
+[Partner Solutions](partner-solutions).
+
+### Piggyback on Partners
+Build a strong commercial and go-to-market framework with our
+[Partner Vendors](partner-solutions) through which we can scale-up products
+that have shown early success with existing customers.
+
+## Product Team
+* Chief Product Officer: Tom Mullen
+* Senior Vice President of Product: Damian Kellly
+* Vice President of Product Engineering: TBC
+
+## What We Believe In
+We believe, above all else, in the following principles, which act as guides to
+help us build incredible products:
+
+**Focus Above the Value Line**
```

Comment:
Under `## Strategy` you use `###` for sub-headers, but in this section you use `**text**`

Reply:
Just style preferences in each of the areas.